fdbserver/Knobs.cpp

```diff
@@ -43,6 +43,7 @@ void ServerKnobs::initialize(bool randomize, ClientKnobs* clientKnobs, bool isSi
 
 	// TLogs
 	init( TLOG_TIMEOUT,                                          0.4 ); //cannot buggify because of availability
+	init( TLOG_SLOW_REJOIN_WARN_TIMEOUT_SECS,                     60 );
```

Comment:
buggify it to like 5 or 10 seconds, so that simulation can trigger the timeout more often.